code/game/machinery/vending_types.dm

```diff
@@ -346,24 +346,24 @@
 		/obj/item/storage/fancy/cigar = 5
 	)
 	prices = list(
-		/obj/item/storage/fancy/cigarettes/rugged = 200,
-		/obj/item/storage/fancy/cigarettes = 245,
-		/obj/item/storage/fancy/cigarettes/dromedaryco = 270,
-		/obj/item/storage/fancy/cigarettes/nicotine = 325,
-		/obj/item/storage/fancy/cigarettes/pra = 230,
-		/obj/item/storage/chewables/rollable/bad = 40,
-		/obj/item/storage/chewables/rollable = 50,
-		/obj/item/storage/chewables/rollable/fine = 60,
-		/obj/item/storage/chewables/rollable/nico = 90,
-		/obj/item/storage/chewables/tobacco/bad = 60,
-		/obj/item/storage/chewables/tobacco = 90,
-		/obj/item/storage/chewables/tobacco/fine = 120,
-		/obj/item/storage/fancy/chewables/tobacco/nico = 150,
+		/obj/item/storage/fancy/cigarettes/rugged = 10,
+		/obj/item/storage/fancy/cigarettes = 15,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 25,
+		/obj/item/storage/fancy/cigarettes/nicotine = 30,
```

Comment:
Again, price should be at least three to four times the price of a pack of chips, about at least 70 and above.

Reply:
Absolutely not.